playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
I'm noticing those includes make the playbook run considerably slower, though I didn't measure time explicitly :cry: 

For the case when there are no checks to be run (and we print a list of available checks), the different is 6s without includes, 8s with includes.